### PR TITLE
Jesus/add product card

### PR DESCRIPTION
**Pull request by jesus-mdz**
Add ProductCard component

### CODE REVIEW

**Review by mmevans — APPROVED** [review]

File: src/pages/ProductCard/ProductCard.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  return (
+    <div className="card m-3">
+      <img
+        style={{ maxWidth: '16rem', maxHeight: '16rem' }}
+        className="card-img-top"
+        src={
+          imageUrl && imageUrl !== ''
+            ? imageUrl
+            : 'https://www.avparty.com/wp-content/uploads/2015/07/image-placeholder-300x300.jpg'
```
Comment (ahaly92) [inline src/pages/ProductCard/ProductCard.js]: can you pull this out into a const at the top of the file

File: src/pages/ProductCard/ProductCard.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        }
+        alt={title}
+      ></img>
+      <div className="card-body text-left">
+        <h5 className="card-title">
+          {title && title !== '' ? title : 'Title'}
+        </h5>
+        <h6 className="card-subtitle mb-2 text-muted justify-content-start">
```
Comment (mmevans) [inline src/pages/ProductCard/ProductCard.js:20]: I've never seen this class before, super cool! (text-muted)

File: src/pages/ProductCard/ProductCard.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+function ProductCard(props) {
+  const { title, price, imageUrl } = props;
+
+  return (
+    <div className="card m-3">
+      <img
+        style={{ maxWidth: '16rem', maxHeight: '16rem' }}
+        className="card-img-top"
+        src={
+          imageUrl && imageUrl !== ''
```
Comment (ahaly92) [inline src/pages/ProductCard/ProductCard.js]: imageUrl check should handle the empty string check

File: src/pages/ProductCard/ProductCard.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            ? imageUrl
+            : 'https://www.avparty.com/wp-content/uploads/2015/07/image-placeholder-300x300.jpg'
+        }
+        alt={title}
+      ></img>
+      <div className="card-body text-left">
+        <h5 className="card-title">
+          {title && title !== '' ? title : 'Title'}
```
Comment (ahaly92) [inline src/pages/ProductCard/ProductCard.js]: same here

**Review by ahaly92 — APPROVED** [review]

File: src/components/Auth/LoginForm.js [hunk 5]
```diff
@@ -4,10 +4,11 @@ import loginActions from '../../store/login/action';
 import { useDispatch } from 'react-redux';
 import loginArray from './const/loginArray';
 
-function LoginForm() {
+function LoginForm(props) {
   const [isSubmitted, setIsSubmitted] = useState('');
   const [allInputsObject, setAllInputsObject] = useState({});
   const dispatch = useDispatch();
+  const { className } = props;
```
Comment (ahaly92) [inline src/components/Auth/LoginForm.js]: you can move this to line 7. function LoginForm({ className }) {

**Pull request merged**